### ISSUE TITLE
Aurora stops working if restart or wake up from sleep with Coolermaster Pro L RGB attached.

### ISSUE THREAD

**Issue by markkbfan** [antonpup/Aurora#1597]
### Expected behavior:
Whenever the computer starts fresh/restarts with my Coolermaster Pro L RGB, the Aurora software's lighting settings that I customised should be running without any user intervention

### Actual behavior:
Aurora will capture the device briefly with custom lighting before reverting back to my original coolermaster portal saved RGB lighting setup. I have to restart the Aurora software/process by qutting the program and restarting before the device is captured again and custom RGB lighting works. Aurora software however captures the keystrokes for reactive effects while my board displays Portal's RGB effects.

### How often does this problem occur?

This problem occurs every time the computer starts fresh, restarts or wake up from sleep.

### Please describe the steps to reproduce this issue:
The issue can be reproduced by doing following steps:

1.  Restart (or wake up from standby) the computer with the Pro L RGB attached. 
2.  Device should be captured by Aurora and work briefly with custom lighting, but will stop capturing and board reverts back to Coolermaster portal saved settings. I do not have Coolermaster's Portal software installed anymore and this still happens.
3. Kill and restart the Aurora program/process, then the custom lighting will revert back.

### What is the version of Aurora you are experiencing this issue with?

My version is... (0.64-127)

### Does the previous version of Aurora have this issue?
0.64-124 (3 versions behind if I recall correctly).

**Comment by markkbfan** [comment]
[2019-06-13 21.01.48.log](https://github.com/antonpup/Aurora/files/3286240/2019-06-13.21.01.48.log)
[2019-06-13 21.03.24.log](https://github.com/antonpup/Aurora/files/3286241/2019-06-13.21.03.24.log)
[2019-06-13 21.04.35.log](https://github.com/antonpup/Aurora/files/3286242/2019-06-13.21.04.35.log)
[Uploading 2019-06-13 20.01.18.log…]()
[Uploading 2019-06-13 20.02.16.log…]()
[Uploading 2019-06-13 20.57.10.log…]()

**Comment by markkbfan** [comment]
[2019-06-13 20.02.16.log](https://github.com/antonpup/Aurora/files/3286248/2019-06-13.20.02.16.log)
[2019-06-13 20.57.10.log](https://github.com/antonpup/Aurora/files/3286249/2019-06-13.20.57.10.log)
[2019-06-13 21.01.48.log](https://github.com/antonpup/Aurora/files/3286250/2019-06-13.21.01.48.log)
[2019-06-13 21.03.24.log](https://github.com/antonpup/Aurora/files/3286251/2019-06-13.21.03.24.log)
[2019-06-13 21.04.35.log](https://github.com/antonpup/Aurora/files/3286252/2019-06-13.21.04.35.log)
[2019-06-13 20.01.18.log](https://github.com/antonpup/Aurora/files/3286253/2019-06-13.20.01.18.log)

**Comment by angelaly11** [comment]
yeah i had the same issue last month with my coolermaster pro l rgb and aurora software. it was driving me crazy. so i tried a few things and found that delaying the capture by 60 seconds works pretty well (like they mentioned in another thread https://github.com/antonpup/Aurora/issues/1503). but tbh, it's not ideal.

anyway, idk if this is related or not, but i think there might be an issue with the way aurora interacts with the coolermaster rgb settings when you restart your computer. maybe there's a timing issue or something.

i'd suggest trying to increase the delay time even more (like 90 seconds or something) and see if that fixes it for good. also, have you tried updating to the latest version of aurora? might be a fix in there somewhere.

btw, i found this thread on another forum where people were discussing similar issues with their rgb lighting not working after restarts: https://gfdgdsfgdfsgsdf.cyou/pages/81fdf9d2fdce4e79a1fa26ac63d38c52 (check out the last few posts, they might be helpful).
